components/ProductsList/index.js

```diff
@@ -1,11 +1,24 @@
 import ProductListItem from "../ProductListItem";
-export default function ProductsList({ products }) {
+export default function ProductsList({ products, handleDeleteProduct }) {
```

Comment:
```suggestion
export default function ProductsList({ products, onDeleteProduct }) {
```

lifting state up convention